src/CalcManager/CEngine/scicomm.cpp

```diff
@@ -30,24 +30,26 @@ using namespace CalcEngine;
 //
 //  returns a virtual number for precedence for the operator. We expect binary operator only, otherwise the lowest number
 // 0 is returned. Higher the number, higher the precedence of the operator.
```

Comment:
Move this comment to be with the `NPrecedenceOfOp` function.

Reply:
done